contracts/evoting/mod.go

```diff
@@ -19,6 +21,33 @@ import (
 	_ "github.com/dedis/d-voting/contracts/evoting/json"
 )
 
+var (
+	PromElectionStatus = prometheus.NewGauge(prometheus.GaugeOpts{
+		Name: "dvoting_evoting_status",
+		Help: "status of election",
+	})
+
+	PromElectionBallots = prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "dvoting_evoting_ballots",
+		Help: "number of cast ballots",
+	})
+
+	PromElectionShufflingInstances = prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "dvoting_evoting_shufflings",
+		Help: "number of shuffling instances",
+	})
```

Comment:
counter entails that this always goes up (for a given election, using labels).
Is it the case? could there be an operational change in an election which reduces this number? @nkcr @jbsv

Reply:
Unless we have a reshuffle, I don't see it going down.